src/garlic-toast.md

```diff
@@ -1,28 +1,28 @@
-# Garlic Toast
-
-Garlic toast, perfect as a substitute for crackers in a cheese platter.
-
-- ⏲️ Prep time: 5 min
-- 🍳 Cook time: 10-15 min
-
-## Ingredients
-
-- Any kind of firm, thin, sliced bread (Sourdough, Baguette)
-- Garlic-infused olive oil (Optionally substitute butter or regular olive oil)
-- Garlic salt
-
-## Directions
-
-1. Preheat oven to 180°C (350°F).
-2. Place sliced bread on a baking tray, covered with aluminium foil
-3. Generously coat bread slices with garlic-infused olive oil.
-4. Season with garlic salt, to taste.
-5. Bake for 10-15 minutes or until crunchy throughout.
-6. Serve with cheese, charcuterie, pickles, dips, or to accommodate a soup. 
-
-## Contribution
-
-- Jack Crawford - [website](https://github.com/jjcrawford)
-- Edited by Zyansheep
-
-;tags: basic bread quick side
+# Garlic Toast
```

Comment:
What changed in this file? not really sure what happened with this diff

Reply:
dos->unix

Reply:
k good to know. Don't think it says anywhere